dist/api_data.py

```diff
@@ -493,6 +493,9 @@ def __ge__(self, other):
     Config('debug_mode', '', r'''
         control the settings of various extended debugging features''',
         type='category', subconfig=[
+        Config('abort_on_corruption_in_diagnostic', 'true', r'''
```

Comment:
That is really long and trying to remember all the words in the right order is hard. How about something like `corruption_abort` instead?